extra-dev/packages/coq-quickchick/coq-quickchick.dev/opam

```diff
@@ -8,13 +8,12 @@ bug-reports: "https://github.com/QuickChick/QuickChick/issues"
 license: "MIT"
 
 build: [
-  [make "compat"]
   ["dune" "subst"] {pinned}
   ["dune" "build" "-p" name "-j" jobs "@install" "@runtest" {with-test} "@doc" {with-doc}]
 ]
 
 depends: [
-  "dune" {>= "2.8"}
+  "dune" {>= "3.12" & < "3.14.2"}
```

Comment:
```suggestion
  "dune" {>= "3.12" & < "3.14.1"}
```
So we know that it has to do with a recent Dune release, and it has to be either 3.14.0 or 3.14.1. Let's establish with certainty which one.